### ISSUE TITLE
NeoTech steel dust smelting might yield the disabled mekanism steel ingot

### ISSUE THREAD

**Issue by Janibek6119** [FTBTeam/FTB-Modpack-Issues#5486]
### Modpack

FTB NeoTech

### Modpack version

1.4.2

### Has the pack been modified

No

### Describe the bug

Smelting MI (Modern Industrialization) Steel dust in Mekanism Smelting Factory yields mekanism steel ingot (which was apparently disabled & hidden in this modpack) instead of MI steel ingot. Meanwhile MI furnaces yield MI steel ingot as expected.

### Steps to reproduce

1. Have a MI Steel dust
2. Smelt it in Mekanism Smelting factory

### Expected behaviour

Smelting MI Steel dust must yield MI Steel ingot

### Screenshots

![image](https://github.com/user-attachments/assets/92c5251d-81a1-4752-832c-c11ed43832af)

Conflicting identical recipes must be the cause:
![image](https://github.com/user-attachments/assets/7e53fa85-74c4-4470-a900-e02acca95a95)

### Additional information

Can be resolved through `/kubejs/server_scripts/recipe_removals.js`? I tried, it worked
```diff
const removeSmeltingByID = [
  "mekanism:processing/iron/ingot/from_dust_smelting",
+  "mekanism:processing/steel/ingot/from_dust_smelting",
```

**Comment by AStickyWicket** [comment]
This is already in game, your kubeJS didn't load properly. In future if you notice this type /reload

**Comment by Janibek6119** [comment]
@AStickyWicket Sorry I'm not local but it doesn't seem to be true. Here's why:

I repeated `/reload`+smelt 10 times. 4 iterations the smelter gave the wrong ingots and 6 it was the right one (MI). Probably it goes 50/50 random after each reload. And 10/10 times EMI recipes still showed two conflicting variants of steel dust smelting.

And I searched `kubejs/` folder and there was no mention of `mekanism:processing/steel/ingot/from_dust_smelting` recipe, and the item `mekanism:ingot_steel` only undergoes that `JEIEvents.hideItems` part and the one where it's stripped of all tags

Also:
- Restarting the game itself (3 times) didn't seem to have any `/reload` effect
- `New instance` & `Repair Instance` didn't bring any difference either
